ops/scripts/utility/check-env-hashes.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+# A utility script useful for identifying deployed branch environments
+
+# Usage
+#   From the root directory, run the following command:
+#     ./ops/scripts/utility/check-env-hashes.sh [-l|h|r|e {hash}]
+
+############################################################
```

Comment:
We could do that too if needed. This was in support of me figuring out which of the existing environments might need to still exist, and I think still adds value in that it would help us identify old environments, even if we add handling of deleted branches, just in case someone forgets to delete their branch.

Reply:
I can throw a commit in for that on the cleanup workflow. it's a one or two line change